client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -469,7 +474,9 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 		if ( themeHasBundle ) {
 			plan = 'business-bundle';
 		} else if ( selectedDesign?.is_externally_managed ) {
-			plan = ! isExternallyManagedThemeAvailable ? PLAN_BUSINESS_MONTHLY : '';
+			const businessPlan =
+				selectedMarketplaceProduct.product_term === 'year' ? PLAN_BUSINESS : PLAN_BUSINESS_MONTHLY;
```

Comment:
This works great when the user has a yearly explorer plan, but not when they have a two year explorer plan (or a three year plan)

Reply:
Yeah, this makes sense, it should be fixed here https://github.com/Automattic/wp-calypso/pull/87312/commits/5235c652e86f54cc28c5d375f672322756397507